tuple/sqlx/tuple_sketch_int64_a_not_b.sqlx

```diff
@@ -21,6 +21,19 @@ config { hasOutput: true }
 
 CREATE OR REPLACE FUNCTION ${self()}(sketchA BYTES, sketchB BYTES)
 RETURNS BYTES
+OPTIONS (
+  description = '''Computes a sketch that represents the scalar set difference of sketchA and not sketchB.
```

Comment:
I think that "scalar" in this context is confusing

Reply:
Removed.